MHIdentityKit/OAuth2/Flows/AuthorizationCodeGrantFlow.swift

```diff
@@ -157,6 +157,17 @@ open class AuthorizationCodeGrantFlow: AuthorizationGrantFlow {
         self.networkClient.perform(request, completion: completion)
     }
     
+    @available(iOS 13, tvOS 13.0.0, macOS 10.15, *)
+    open func perform(_ request: URLRequest) async -> NetworkResponse {
```

Comment:
This implementation not needed, because it is handled by the `AuthorizationGrantFlow` extension.